themes/openy_themes/openy_carnation/templates/form/form--groupex-form-full.html.twig

```diff
@@ -0,0 +1,26 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a 'form' element.
+ *
+ * Available variables
+ * - attributes: A list of HTML attributes for the wrapper element.
+ * - children: The child elements of the form.
+ *
+ * @see template_preprocess_form()
+ *
+ * @ingroup themeable
+ */
+#}
+<script type="text/javascript">(function () {
+  if (window.addtocalendar)if(typeof window.addtocalendar.start == "function")return;
+  if (window.ifaddtocalendar == undefined) { window.ifaddtocalendar = 1;
+    var d = document, s = d.createElement('script'), g = 'getElementsByTagName';
+    s.type = 'text/javascript';s.charset = 'UTF-8';s.async = true;
+    s.src = ('https:' == window.location.protocol ? 'https' : 'http')+'://addtocalendar.com/atc/1.5/atc.min.js';
```

Comment:
let's attach library via Drupal style instead of hardcoding external service code
see https://www.drupal.org/node/2456753

Reply:
@podarok Sounds good, this is also in the rose & lily theme, would you like me to make this a global change in both those themes as well?

Reply:
@aaronjchristian yes, please.